charts/stable/pod-gateway/templates/webhook-deployment.yaml

```diff
@@ -50,7 +50,7 @@ spec:
             # Static
             - --tls-cert-file-path=/tls/tls.crt
             - --tls-key-file-path=/tls/tls.key
-            - --DNSPolicy=None
+            - --DNSPolicy=DNS={{ .Values.DNSPolicy }}
```

Comment:
Shouldn't this be `- --DNSPolicy={{ .Values.DNSPolicy }}` ?

Reply:
Oops. Totally. That was a failed paste! Fixed now!